conf/modules_alphafold2.config

```diff
@@ -0,0 +1,61 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args   = Additional arguments appended to command in module.
+        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix = File name prefix for output files.
+----------------------------------------------------------------------------------------
+*/
+
+//
+// General configuration options
+//
+
+process {
+    withName: 'GUNZIP|COMBINE_UNIPROT|DOWNLOAD_PDBMMCIF' {
+        publishDir = [
+            path: {"${params.outdir}/DBs/${params.mode}/${params.alphafold2_mode}"},
+            mode: 'symlink',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+        ]
+    }
+}
+
+if (params.alphafold2_mode == 'standard') {
+    process {
+        withName: 'RUN_AF2' {
+            ext.args   = params.use_gpu ? '--use_gpu_relax=true' : '--use_gpu_relax=false'
+            publishDir = [
+                path: { "${params.outdir}/${params.mode}/${params.alphafold2_mode}" },
+                mode: 'copy',
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+            ]
+        }
+    }
+}
+
+
+if (params.alphafold2_mode == 'split_msa_prediction') {
+    process {
+        withName: 'RUN_AF2_MSA' {
+            ext.args =  params.max_template_date ? "--max_template_date ${params.max_template_date}" : ''
```

Comment:
Could the params.max_template_date be passed using the ext.args2 in the RUN_AF2 module?

Reply:
Fixed here [14d13f0](https://github.com/nf-core/proteinfold/pull/61/commits/14d13f05a6cd16be2c72e95cdbd27b23f069a093)